packages/api-explorer/src/components/DocPseudo/DocParam.tsx

```diff
@@ -11,6 +36,6 @@ export const DocParam: FC<DocArgProps> = ({ parameter }) => (
     content={`${parameter.type.name} ${parameter.description}`}
     placement="bottom"
   >
-    <span>{parameter.required ? parameter.name : `[${parameter.name}]`}</span>
+    {parameter.required ? parameter.name : `[${parameter.name}]`}
```

Comment:
🥂

Reply:
It seems like tooltips are not showing anymore.

Reply:
I added spans back